tests/google/cloud/operators/test_bigquery.py

```diff
@@ -142,37 +143,12 @@ def test_bigquery_insert_job_operator_execute_complete():
     mock_log_info.assert_called_with("%s completed with response %s ", "insert_query_job", "Job completed")
 
 
-@mock.patch("airflow.providers.google.cloud.operators.bigquery.hashlib.md5")
-@pytest.mark.parametrize(
-    "test_dag_id, expected_job_id",
-    [("test-dag-id-1.1", "airflow_test_dag_id_1_1_test_job_id_2020_01_23T00_00_00_00_00_hash")],
-    ids=["test-dag-id-1.1"],
-)
-def test_job_id_validity(mock_md5, test_dag_id, expected_job_id):
```

Comment:
We should be able to test that `self.hook.generate_job_id(` is called with correct arguments

Reply:
Ok Will add a test case

Reply:
Added test case